src/main/java/com/icthh/xm/tmf/ms/communication/service/mail/EmailTemplateService.java

```diff
@@ -44,4 +57,29 @@ public RenderTemplateResponse renderEmailContent(RenderTemplateRequest renderTem
             throw new RenderTemplateException(e.getMessage(), renderTemplateRequest.getContent(), renderTemplateRequest.getModel());
         }
     }
+    @SneakyThrows
+    public void updateTemplate(String templateKey, UpdateTemplateRequest updateTemplateRequest) {
+        EmailSpec emailSpec = emailSpecService.getEmailSpec();
+
+        EmailTemplateSpec emailTemplateSpec = emailSpec.getEmails().stream()
```

Comment:
this code it must be method in emailSpecService.getEmailSpecByTemplateKey(String templateKey)

Reply:
moved to method